suite.go

```diff
@@ -6,28 +6,33 @@ import (
 	"testing"
 
 	"github.com/docker/docker/client"
+	"github.com/docker/docker/daemon/logger"
 )
 
 func init() {
 	registry = make(map[string]*Suite)
 }
 
 var registry map[string]*Suite
+var logWatcher *logger.LogWatcher
+var logDone = make(chan bool, 1)
 
 // SuiteOpts is an option struct for getting or creating a suite in GetOrCreateSuite.
 type SuiteOpts struct {
 	// optional docker client, if one already exists
 	Client *client.Client
 	// whether to fail on instantiation errors
-	Skip bool
+	Skip      bool
+	Verbosity bool
```

Comment:
`Verbosity` sounds like it has multiple levels, but in fact it can be `Verbose` or not.